### PR TITLE
docs: first round of comment correction and extensions

### PR DESCRIPTION
**Pull request by TotalPizza**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [X] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [X] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

### CODE REVIEW

**Comment by TotalPizza** [comment]
Took the liberty to change functions that call `Stack.pop` multiple times to `Stack.pop_n`

File: src/kakarot/instructions.cairo [hunk 1]
```diff
@@ -33,8 +33,9 @@ from kakarot.instructions.sha3 import Sha3
 // @custom:namespace EVMInstructions
 namespace EVMInstructions {
     // @notice Decode the current opcode and execute associated function.
+    // @dev The function iterrates through the provided instructions and executed each of them
```
Comment (Eikix) [inline src/kakarot/instructions.cairo:36]: iterates*
executes*

**Review by Eikix — DISMISSED** [review]

File: src/kakarot/instructions.cairo (hunk 1 above)
Comment (TotalPizza) [reply src/kakarot/instructions.cairo]: Corrected

**Review by Eikix — APPROVED** [review]

**Comment by Eikix** [comment]
Going to merge this PR as first part of clarification of codebase and integration of comments

**Pull request merged**